.gitignore

```diff
@@ -1,6 +1,3 @@
-# source code fetched by the build
-/nginx/
-
```

Comment:
nginx is now fetched by cmake into the build dir so it works on mac os. The problem now is that nginx is configured on mac os, but then when it runs in docker it's on linux, so the build fails

Reply:
Ah, I made a recent change in build #66. Now, CMake expects the NGINX source directory as input, specified with `-DNGINX_SRC_DIR`, and configures accordingly. Instead of downloading the source code each time, which isn't ideal given that I work with both the stable and mainline NGINX branches, there are a few advantages to this approach:

1. It's quicker to simply rename the folder or adjust the project configuration to use a different directory rather than waiting for the script to download.
2. By avoiding unnecessary downloads every-time consider it should re-configure the project, I'm contributing to saving the planet.
3. I'm not aware of a .tar.gz file following the HEAD of NGINX.

At head it should work on MacOS. If not, that's something we can discuss.